compiler/sigmatch.nim

```diff
@@ -1650,7 +1650,7 @@ proc matchesAux(c: PContext, n, nOrig: PNode,
     if a >= formalLen-1 and formal != nil and formal.typ.isVarargsUntyped:
       incl(marker, formal.position)
       if container.isNil:
-        container = newNodeIT(nkBracket, n.sons[a].info, arrayConstr(c, n.info))
+        container = newNodeIT(nkArgList, n.sons[a].info, arrayConstr(c, n.info))
```

Comment:
Well, these nodes never reach the backend. They are created only to be passed in the macro call, which will be replaced a bit later by its AST result. One can imagine a silly macro that just returns the input AST, but my money would be that this would trigger malformed AST error in sem.